config/dynamic.c

```diff
@@ -213,13 +213,84 @@ read_nbytes(int conn_fd, char *msg_buff, int nbytes)
 	return 0;
 }
 
+static int
+process_return_message(int conn_fd, struct dynamic_config *dy_conf)
+{
+	int i;
+	bool cond = false;
+	int loop = 0;
+	/* The maximum number of times to try to obtain the GT return messages. */
+	const int MAX_NUM_RETURN_TRY = 3;
+	int num_cmd;
+	struct dy_cmd_entry *dy_cmds[dy_conf->mailbox_burst_size];
+	int num_gt_messages = 0;
+	size_t reply_len = 0;
+	char reply_msg[MSG_MAX_LEN];
+
+	while (!cond && loop < MAX_NUM_RETURN_TRY) {
```

Comment:
1. We can drop variable `cond` and replace it with the test `num_gt_messages < dy_conf->gt->num_lcores` here.

2. The test `loop < MAX_NUM_RETURN_TRY` is not reasonable to deal with the synchronization between the dynamic block and the GT blocks because it's vulnerable to the running time of the update code. An alternative must be investigated.